go.mod

```diff
@@ -4,6 +4,7 @@ go 1.22.6
 
 require (
 	github.com/brianvoe/gofakeit/v7 v7.0.4
+	github.com/dgrijalva/jwt-go v3.2.0+incompatible
```

Comment:
This is dprecated. Please use https://github.com/golang-jwt/jwt

Reply:
Done